core/mask_rcnn/mrcnn/model.py

```diff
@@ -1254,8 +1258,10 @@ def hook(images, augmenter, parents, default):
         mask = det.augment_image(mask.astype(np.uint8),
                                  hooks=imgaug.HooksImages(activator=hook))
         # Verify that shapes didn't change
-        assert image.shape == image_shape, "Augmentation shouldn't change image size"
-        assert mask.shape == mask_shape, "Augmentation shouldn't change mask size"
```

Comment:
This might be a valid use of `assert`?

Reply:
Agreed, do you think it's best to leave it as it was and add a `# nosec` comment?

Reply:
Or, we can raise an error as other changes if the check fails. Either works for me.

Reply:
A `#nosec` should work